app/assets/stylesheets/style.css

```diff
@@ -0,0 +1,238 @@
+body {
+  background-color: rgba(240, 245, 245, 0.5);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style-type: none;
+  font-size: 16px;
+  font-family: 'Roboto', sans-serif;
+}
+
+.group_card {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: white;
+  margin: 24px;
+  border-radius: 24px;
+  padding: 12px;
```

Comment:
- [ ] To match the given design, kindly remove the border-radius on the categories and transaction cards. The font-weight does not match the original, kindly fix this issue as well.

![cat](https://github.com/chepkok3/Budget-app/assets/53124548/2ddc68d5-dc15-41f1-83b1-92a865d1ce13)

## Original

![og](https://github.com/chepkok3/Budget-app/assets/53124548/a87b0ffa-0399-46ed-b4aa-0b6ee3247ea7)